server_apps/DB_maintenance/validatedata/Check-Fish-Name-Function-Mismatch_d.sql

```diff
@@ -1,5 +1,20 @@
-select fish_zdb_id, fish_name, get_fish_name(fish_zdb_id) as computed_fish_name from fish
-where  fish_name <> get_fish_name(fish_zdb_id)
-and fish_name <> 'Cooch Behar' -- exception
-order by fish_zdb_id
-
+SELECT
+    fish_zdb_id,
+    fish_name,
+    get_fish_name (fish_zdb_id) AS computed_fish_name
+FROM
+    fish
+WHERE (
+            fish_name <> get_fish_name (fish_zdb_id)
+        OR trim(fish_name) = ''
+        OR trim(get_fish_name (fish_zdb_id)) = ''
+        OR trim(fish_name) = '(AB)' -- basically a blank name, but with a background
```

Comment:
What about other background gentoypes, ie other than 'AB', e.g. 'TL'?

Reply:
Good point. I updated it to look for names that start with '('.